client/context/loadbalancing.go

```diff
@@ -0,0 +1,46 @@
+package context
```

Comment:
Seems like this entire needs to be formatted correctly. Also, it should probably be renamed to `client_manager.go`